.github/workflows/doc.yml

```diff
@@ -23,14 +23,22 @@ jobs:
           rustup toolchain list -v
           # Find toolchain path
           export TOOLCHAIN=$(rustup toolchain list -v | grep active | awk '{print $NF}')
-          echo "toolchain=$TOOLCHAIN"
+          echo "toolchain=\"$TOOLCHAIN\""
           # Remove rustc book
           rm $TOOLCHAIN/share/doc/rust/html/rustc -rf
           # Download rustc API docs
           rustup component add rustc-docs
           # Move the docs to deployment path
           mv $TOOLCHAIN/share/doc/rust/html/rustc charon/target/doc/rustc
 
+      - name: Prepare index.html
+        run: |
+          # Find toolchain name
+          export toolchain=$(rustup toolchain list | grep active | grep -oP "nightly-\d{4}-\d{2}-\d{2}")
```

Comment:
Same here, the toolchain name can be found in the `./rust-toolchain` toml file. Try:
```bash
cargo install toml2json
toolchain="$(toml2json rust-toolchain | jq -r .toolchain.channel)"
```

Reply:
Would you mind using https://github.com/taiki-e/install-action?tab=readme-ov-file#supported-tools to install toml2json? The action saves installing it from compiling from source code, it may download toml2json from github via cargo-binstall instead.

Reply:
Or rather `rustc --print sysroot | grep -oP "nightly-\d{4}-\d{2}-\d{2}"` just works.